src/Pretzel.Tests/Templating/Jekyll/LiquidEngineTests.cs

```diff
@@ -787,8 +787,8 @@ public override LiquidEngine Given()
 
             public override void When()
             {
-                FileSystem.AddFile(@"C:\website\_site\BadFormat.md", new MockFileData(OriginalPageContents) { LastWriteTime = new DateTime(2010, 01, 2) });
-                FileSystem.AddFile(@"C:\website\BadFormat.md", new MockFileData(PageContents) { LastWriteTime = new DateTime(2010, 01, 3) });
+                FileSystem.AddFile(@"C:\website\_site\BadFormat.md", new MockFileData(OriginalPageContents) { LastWriteTime = new DateTime(2013, 01, 2) });
```

Comment:
I don't know. It may have come from some silly idea. I have cherry picked this commit from a dormant branch I was using since about half a year ago.